client/src/client-only-routes/ShowProjectLinks.js

```diff
@@ -173,23 +179,22 @@ const ShowProjectLinks = props => {
           solution={solution}
         />
       ) : null}
-      If you suspect that any of these projects violate the{' '}
-      <a
-        href='https://www.freecodecamp.org/news/academic-honesty-policy/'
-        target='_blank'
-        rel='noreferrer'
-      >
-        academic honesty policy
-      </a>
-      , please{' '}
-      <a
-        href={`/user/${username}/report-user`}
-        target='_blank'
-        rel='noreferrer'
-      >
-        report this to our team
-      </a>
-      .
+      <Trans i18nKey='certification.project.footnote'>
```

Comment:
```suggestion
      <Trans i18nKey='certification.project.footnote'>
        If you suspect that any of these projects violate the{' '}
```
To provide the default.

Reply:
Weird but after adding this to provide default value react-i18next does not pick up the nested values and renders nothing in place of the `a` tags

![Screenshot from 2021-02-26 01-11-04](https://user-images.githubusercontent.com/64462998/109208013-ce6b3480-77cf-11eb-8ffd-741bcc77a937.png)

![Screenshot from 2021-02-26 01-10-42](https://user-images.githubusercontent.com/64462998/109208000-cc08da80-77cf-11eb-9a7c-b18668272391.png)

Reply:
Weird but after adding this to provide default value react-i18next renders nothing in place of the `a` tags

![Screenshot from 2021-02-26 01-11-04](https://user-images.githubusercontent.com/64462998/109208013-ce6b3480-77cf-11eb-8ffd-741bcc77a937.png)

![Screenshot from 2021-02-26 01-10-42](https://user-images.githubusercontent.com/64462998/109208000-cc08da80-77cf-11eb-9a7c-b18668272391.png)